<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>664</width>
         <height>318</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Приложение</string>
    </property>
    <addaction name="file_action"/>
    <addaction name="dir_action"/>
    <addaction name="separator"/>
    <addaction name="update"/>
    <addaction name="history"/>
    <addaction name="separator"/>
    <addaction name="close_action"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Установить размер иконок</string>
     </property>
     <addaction name="resize300_500"/>
     <addaction name="resize500_800"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Установить количество столбцов</string>
     </property>
     <addaction name="cln1"/>
     <addaction name="cln2"/>
     <addaction name="cln3"/>
     <addaction name="cln4"/>
    </widget>
    <addaction name="root_dir"/>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="file_action">
   <property name="text">
    <string>Читать из архива</string>
   </property>
   <property name="shortcut">
    <string>Shift+Z</string>
   </property>
  </action>
  <action name="close_action">
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="dir_action">
   <property name="text">
    <string>Читать из папки</string>
   </property>
   <property name="shortcut">
    <string>Shift+F</string>
   </property>
  </action>
  <action name="root_dir">
   <property name="text">
    <string>Установить корневую папку</string>
   </property>
  </action>
  <action name="resize300_500">
   <property name="text">
    <string>300х500</string>
   </property>
  </action>
  <action name="resize500_800">
   <property name="text">
    <string>500х800</string>
   </property>
  </action>
  <action name="cln1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="cln2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="cln3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="cln4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="update">
   <property name="text">
    <string>Обновить меню</string>
   </property>
  </action>
  <action name="history">
   <property name="text">
    <string>История</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
